test/Microsoft.AspNet.Mvc.ModelBinding.Test/Validation/ModelStateDictionaryTest.cs

```diff
@@ -9,6 +9,53 @@ namespace Microsoft.AspNet.Mvc.ModelBinding
 {
     public class ModelStateDictionaryTest
     {
+
+        [Fact]
+        public void MarkFieldSkipped_Throws_IfStateIsInvalid()
```

Comment:
Could we add tests that verify positive scenarios (MarkFieldSkipped_MarksFieldAndSubKeys_Valid)